contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -732,12 +730,23 @@ varbinaryvarchar(PG_FUNCTION_ARGS)
 	VarChar    *result;
 	char 	   *encoded_result;
 	size_t		len = VARSIZE_ANY_EXHDR(source);
-	int32		typmod = PG_GETARG_INT32(1);
-	int32		maxlen = typmod - VARHDRSZ;
+	int32		typmod = -1;
+	int32		maxlen = -1;
 	coll_info	collInfo;
 	int			encodedByteLen;
 	MemoryContext ccxt = CurrentMemoryContext;
 
```

Comment:
3rd argument is not used at all? We should a comment that there are two callers to this function with 1 and 3 arguments. So, if someone parses 3rd argument tomorrow, they should make sure that number of args are checked before parsing